source/common/quic/envoy_quic_proof_verifier.h

```diff
@@ -18,6 +18,36 @@ class CertVerifyResult : public quic::ProofVerifyDetails {
   bool is_valid_{false};
 };
 
+// An interface for the Envoy specific QUICHE verify context.
```

Comment:
nit: QUICHE => QUIC, I think, to match the class name and since this is QUIC specific.